.github/workflows/test.yml

```diff
@@ -10,13 +15,56 @@ jobs:
   # test local kind start up
   test_localkind:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # I suppose we'd better update version pre kepler release.
+        # keep from current default version and latest version.
+        kubectl_version: [v1.25.4, v1.27.2]
+        kind_version: [0.17.0, 0.19.0]
+    steps:
+    - uses: actions/checkout@v3
+    - name: kubectl
+      run: curl -LO https://dl.k8s.io/release/${{matrix.kubectl_version}}/bin/linux/amd64/kubectl
+    - name: start from local
+      run: | 
+        export KIND_VERSION=${{matrix.kind_version}}
+        ./main.sh up
+    - name: verify
+      # todo: 
+      # - make verification in a shell script
+      # - make checking prometheus
+      # - make checking grafana
+      run: |
+        dpkg -l | grep bcc
+        docker ps -a
+        kind get kubeconfig --name=kind
+        kubectl get pods 
+
+# test local kind start up
+  test_localkind_with_prometheus:
+    runs-on: ubuntu-latest
+    needs: test_localkind
+    strategy:
+      matrix:
+        # I suppose we'd better update version pre kepler release.
+        # keep from current default version and latest version.
+        prometheus_operator_version: [v0.11.0, v0.12.0]
+        grafana_enabled: [true, false]
```

Comment:
Why do we need to test with and without grafana?
Actually we don't need to enable grafana for testing in CI.

Reply:
so far I just want to test our shell script to ensure nothing breaking.